msg/takeoff_status.msg

```diff
@@ -0,0 +1,11 @@
+# Status of the takeoff state machine currently just availble for multicopters
+
+uint64 timestamp # time since system start (microseconds)
+
+uint8 TAKEOFF_STATE_DISARMED = 0
+uint8 TAKEOFF_STATE_SPOOLUP = 1
+uint8 TAKEOFF_STATE_READY = 2
+uint8 TAKEOFF_STATE_RAMPUP = 3
+uint8 TAKEOFF_STATE_FLIGHT = 4
```

Comment:
You can then initialize the struct in `takeoff.hpp` with those values to make sure they're always in sync.

Reply:
We already have

	TakeoffState _takeoff_state = TakeoffState::disarmed;

Reply:
The reason to not do this is that the class then depends on the uORB message definition.
But I can change it. Definitely better if it's only defined once.

Reply:
Fixed in d69d31b